fuzzing/tools/BUILD

```diff
@@ -15,10 +15,38 @@
 
 # This package contains tool scripts
 
-load("@rules_python//python:defs.bzl", "py_binary")
+load("@rules_python//python:defs.bzl", "py_binary", "py_library", "py_test")
+
+# Tools.
+########
 
 py_binary(
     name = "launcher",
     srcs = ["launcher.py"],
     visibility = ["//visibility:public"],
 )
+
+py_binary(
+    name = "validate_dict",
+    srcs = ["validate_dict.py"],
+    visibility = ["//visibility:public"],
+    deps = [":dict_validation"],
+)
+
+# Libraries.
+############
+
+py_library(
+    name = "dict_validation",
+    srcs = ["dict_validation.py"],
+)
+
+# Tests.
+########
+
+py_test(
+    name = "dict_validation_test",
+    srcs = ["dict_validation_test.py"],
+    tags = ["test"],
```

Comment:
I would like to understand why this is needed. I feel it is unsustainable to expect everyone to remember to add this tag to their unit tests.

Reply:
I am not sure which may have higher risk, forgetting to add this tag that leads to no CI test on the target or keeping changing the CI test script.

My original opinion is that keeping updating CI test script may cause more severe problem. That impression comes from Github requiring the authentication token to update the CI script, which is not needed in other place.

Reply:
Maybe https://docs.bazel.build/versions/master/command-line-reference.html#flag--build_tests_only will help? I agree that manual tagging will be unsustainable

Reply:
Edit: another thing that can be done is make a placeholder `_test` fule for the "bazel test some_fuzz_test" that will be able to be filled in with a unit test implementation. 

(I think https://github.com/googleinterns/bazel-rules-fuzzing/blob/2cd72332724c1b1a4eca9edbc85a5f46cc60567a/fuzzing/cc_deps.bzl#L36 will have to become a `cc_binary`, so that it is not a `_test` rule, and a seperate `fuzzing_regression_test` rule needs to be made

Reply:
I have a maybe stupid question, why we need `cc_binary` instead of `_test`? Is it just to avoid cases like this?